docs/features.rst

```diff
@@ -79,6 +79,28 @@ directive::
     /:
         inherit: false
 
+
+.. _config:
+
+Config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, all hidden files are ignored when exploring metadata
+on the disk. If a specific file or directory should be included in
+the search, create a simple config file ``.fmf/config`` with the
+following format:
```

Comment:
Probably should move these sections to be together in a fmf control section where all fmf metadata can be found. Currently `.fmf/version`, `.fmf/config`, `/:` configurations, etc.

For the `explore.include` specifically, we should explicitly state that `.fmf` folder cannot be used in there. Currently we technically allow it, but it feels like it will ask for trouble in the future.

Reply:
> Probably should move these sections to be together in a fmf control section where all fmf metadata can be found. Currently `.fmf/version`, `.fmf/config`, `/:` configurations, etc.

Good, point. What about documenting `Trees` and `Config` close together? See ecd8e0f.

> For the `explore.include` specifically, we should explicitly state that `.fmf` folder cannot be used in there. Currently we technically allow it, but it feels like it will ask for trouble in the future.

Yeah, makes sense. Explicitly mentioned and added an error message in ecd8e0f.

Reply:
> > Probably should move these sections to be together in a fmf control section where all fmf metadata can be found. Currently `.fmf/version`, `.fmf/config`, `/:` configurations, etc.
> 
> Good, point. What about documenting `Trees` and `Config` close together? See [ecd8e0f](https://github.com/teemtee/fmf/commit/ecd8e0f0c90dd1c871b6b3f052206bcb235dabb1).

Was thinking more like in a future refactoring because some of these seem like advanced design tools. The restructuring I am thinking of is:
- "What is fmf": a simple explanation of fmf as yaml + file structure with inheritance.
- "Features": all the capabilities and definitions of fmf
- "Advanced designs": suggestions for how to structure the fmf tree, advanced in-tree control parameters like `/.inherit`, advanced tree control parameters in `.fmf/config`, full list of operators (just because can't find a better place)
- "Developer corner": what to expect from `.fmf/version`, how to expand fmf and more such topics

The concept page is too `tmt` specific and it's hard to parse outside of it. I think most of the content could be converted to a "what/why fmf".

---

But this is all for a different PR, I'll create an issue from this comment